source/ДЗ/УсловныеОператоры4/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace УсловныеОператоры4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение Х");
+            var x = double.Parse(Console.ReadLine());
+            Console.WriteLine(FX(x));
+            Console.ReadKey();
+
+        }
+        static double FX(double x)
+        {
+            if (x > 1)
+                return 2 - x;
+            else if (x <= 1 & x >= -1)
```

Comment:
Лишнее логическое выражение. 
```suggestion
            else if (x >= -1)
```
В эту ветку кода мы попадем только если предыдущее логическое выражение будет ложным, т. е. когда _x_ <= -1.